pkg/provider/azure_controller_common.go

```diff
@@ -267,7 +281,46 @@ func (c *controllerCommon) AttachDisk(ctx context.Context, async bool, diskName,
 			klog.Warningf("azureDisk - switch to batch operation due to rate limited, QPS: %f", c.diskOpRateLimiter.QPS())
 		}
 	}
-	return lun, vmset.WaitForUpdateResult(ctx, future, nodeName, "attach_disk")
+
+	err = c.waitForUpdateResult(ctx, vmset, nodeName, future, err)
```

Comment:
`return lun, c.waitForUpdateResult(ctx, vmset, nodeName, future, err)` is ok, it's not necessary to log success since there is already logging outside (in CSI driver)